sdk/digitaltwins/Azure.DigitalTwins.Core/src/autorest.md

```diff
@@ -36,3 +36,10 @@ directive:
 #payload-flattening-threshold: 1
 #namespace: Azure.DigitalTwins.Core
 ```
+
+``` yaml
```

Comment:
This will remove the enum section of the definition and as a result the IfNonMatch will be generated as an option

Reply:
Please add that comment to the file.